lib/suse/connect/api.rb

```diff
@@ -79,6 +83,22 @@ def make_paginated_request(method, url, options = {})
         @entities
       end
 
+      private
+
+      def system_uuid
+        @system_uuid ||= fetch_system_uuid
```

Comment:
It would be better to merge this method with `fetch_system_uuid`, there shouldn't be a need for a 1-line method.